src/selection.ts

```diff
@@ -1,11 +1,22 @@
 import {Binding} from 'vega';
 import {SingleDefChannel} from './channel';
+import {DateTime} from './datetime';
 import {EventStream} from './vega.schema';
 
 export const SELECTION_ID = '_vgsid_';
 export type SelectionType = 'single' | 'multi' | 'interval';
 export type SelectionResolution = 'global' | 'union' | 'intersect';
 
+export type SelectionInit = boolean | number | string | DateTime;
+export type SelectionInitArray = boolean[] | number[] | string[] | DateTime[];
+export interface SelectionInitMapping {
+  [key: string]: SelectionInit;
+}
+
+export interface SelectionInitArrayMapping {
+  [key: string]: SelectionInitArray;
```

Comment:
The key here is either a field or a channel, right? Can we make that explicit?

Reply:
Sorry it doesn't work, which means we can merge at this point. 

![image](https://user-images.githubusercontent.com/111269/53292459-a297bd00-3777-11e9-9f70-1f3fe80e9afc.png)

Reply:
Well, was an idea.